CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to Dynamico
+
+Welcome, and thank you for your interest in contributing to Dynamico!
+
+There are many ways in which you can contribute, beyond writing code. The goal of this document is to provide a high-level overview of how you can get involved.
+
+## Asking Questions
+
+Have a question? Rather than opening an issue, please ask away on [Stack Overflow](https://stackoverflow.com/) using the a related tag
+
+The active community will be eager to assist you. Your well-worded question will serve as a resource to others searching for help.
+
+## Providing Feedback
+
+Your comments and feedback are welcome, and the development team is available via a handful of different channels:
+
+- GitHub issues
+- Any related slack channels or stackoverflow
+
+## Reporting Issues
+
+Have you identified a reproducible problem in Dynamico? Have a feature request? We want to hear about it! Here's how you can make reporting your issue as effective as possible.
+
+### Look For an Existing Issue
+
+Before you create a new issue, please do a search in (https://github.com/Soluto/dynamico/issues) to see if the issue or feature request has already been filed.
```

Comment:
The link is merged with the word next to it resulting in a broken link. 
<img width="356" alt="Screen Shot 2022-07-10 at 22 50 33" src="https://user-images.githubusercontent.com/71976177/178160006-84ac7f96-bed0-4293-b4fc-2ab452535528.png">
We can solve it by replacing the space character.

And let's make the link more readable:
```suggestion
Before you create a new issue, please do a search in [open issues](https://github.com/Soluto/dynamico/issues) to see if the issue or feature request has already been filed.
```